include/tvm/tir/schedule/schedule.h

```diff
@@ -299,6 +302,20 @@ class Schedule : public runtime::ObjectRef {
    */
   TVM_DLL static Schedule Concrete(IRModule mod, int debug_mode,
                                    ScheduleErrorRenderLevel error_render_level);
+  /*!
+   * \brief Construct a traced concrete TensorIR schedule from an IRModule
+   * \param mod The IRModule to be scheduled
+   * \param debug_mode Do extra correctness checking after the class creation
+   * and each time after calling the Replace method.
+   * \param error_render_level The level of error rendering
+   * \return The concrete schedule created
+   * \sa ScheduleDebugMask
+   * \note The checks performed include:
+   * 1) VerifySRefTree
+   * 2) VerifyCachedFlags
+   */
+  TVM_DLL static Schedule Traced(IRModule mod, int debug_mode,
```

Comment:
ScheduleDebugMask is a uint32_t, yet `debug_mode` is an int. Shouldn't they be the same type?

Reply:
Also the name `debug_mode` implies that this is an integral value, when it is in fact a mask. Could we use something like `debug_mask` instead?

Reply:
Yeah it definitely sounds better to use debug_mask. Will do